core/src/test/java/com/graphhopper/routing/util/TagParsingTest.java

```diff
@@ -128,37 +128,6 @@ public void testMixBikeTypesAndRelationCombination() {
         assertTrue(bikePriorityEnc.getDecimal(false, edgeFlags) > mtbPriorityEnc.getDecimal(false, edgeFlags));
     }
 
-    @Test
-    public void testCompatibilityBug() {
-        EncodingManager manager2 = EncodingManager.create("bike2");
-        ReaderWay osmWay = new ReaderWay(1);
```

Comment:
Not sure if this test is really needed but for #2669 I just replaced bike2 with mtb...